app/assets/stylesheets/administrate/components/_buttons.scss

```diff
@@ -34,3 +34,8 @@
     }
   }
 }
+
+.scope-buttons {
+  margin: 3px;
+  padding-left: 1.5em;
```

Comment:
The two values above for `margin` and `padding` feel like they don’t have any meaning, or relate to anything. Where does `3px` and `1.5em` come from? Why use different units?